docs/development.rst

```diff
@@ -868,38 +882,82 @@ Next, the species.py file will need to be edited with the species-specific infor
+the existing species.py files inside the catalog/ directory for reference.
 
------------------------------------
-Tests for new species
------------------------------------
+Once these fields have been entered, you should be able to load and simulate
+the newly added species using `stdpopsim`.
 
-Basic sanity tests for the new species will be completed through QC over in the `tests/test_{species_id}.py` file, which was also created by the add-species utility. To run the tests in stdpopsim we use the pytest module.
+However, the species still needs to be checked by someone else, i.e, QC'ed.
+The maintenance script has also created a minimal set of tests in the file
+`tests/test_{species_id}.py`. These tests should *not* be filled out by the
+person who adds the species, but rather by someone else, as part of the
+review process. However, some tests are already present,
+and you can run them as follows:
 
 .. code-block:: shell
 
    python -m pytest tests/test_AnoGam.py
 
-This test checks for things related to missing information and formatting. For example, it wants the citation year to be of type `int` rather than `str` (i.e. no quotes).
+This will check for things related to missing information and formatting. For
+example, it checks that the citation year is of type `int` rather than `str`
+(i.e., no quotes).
+
+At some point during adding a species, you should start a pull request with your
+changes. It does not have to be finished, since it's much easier for others to
+help if they can see your code.
 
 ----------------------
 Species review process
 ----------------------
-Once you are satisfied that the species can be simulated via the CLI, submit a pull
-request with your changes. The species definition will go through a review process.
-This process includes not only a code review, but also includes a QC process to double
-check parameters and citations are appropriate.  To initiate the QC process, open a
+
+Once everything works, we will merge your pull request, and the species will be
+in stdpopsim, but still needs to go through a review process, in which someone else
+checks parameters and citations as appropriate.  To initiate this process, open a
 new `issue <https://github.com/popsim-consortium/stdpopsim/issues/new/choose>`__
 using the 'Species QC issue template'. One or more volunteers will check items off
 the checklist, until all items have been completed satisfactorily. The QC issue,
-or the pull request, may be used for review discussion. The new species will be
-merged once the checklist is completed.
+or the pull request, may be used for review discussion.
+
+To begin reviewing (i.e., QC'ing) a species, you should state your intention on the
+QC issue (so we don't duplicate effort) and start a pull request to fill out the code.
+During the process of QC, other developers fill in the tests that are disabled
+in the `tests/test_{species_name}.py` file. For instance, a test for an
```

Comment:
Maybe mention here that the person doing the QC should be looking up the parameters in the provided citations, without looking at the values provided by the original submitter (I think that's how it works)?